### ISSUE TITLE
Add L10n support to subscription lists

### ISSUE THREAD

**Issue by myrdd** [RequestPolicyContinued/requestpolicy#550]
([L10n = Localization](https://en.wiktionary.org/wiki/L10n))

The [official subscription list](https://github.com/RequestPolicyContinued/subscriptions/blob/master/official.json) – see the [subscription repo](https://github.com/RequestPolicyContinued/subscriptions/) – will soon contain **translations** of the subscriptions' titles and descriptions.

It's **yet to be decided** where the L10n strings will be put. I think there are three possibilities
1. _per-language_ files (`N` languages ⇒ `N` files)
2. put all L10n strings in _one_ file
3. put all L10n strings in `official.json` (no separate file)

We have to find the **pros** and **cons** of those solutions. Some aspects might be:
- the amount of kilobytes _a single user_ has to download
- the frequency how often the L10n file is changed
  - to avoid unnecessary downloads, `official.json` might include the date of the last change, just like with the „serials“ of the subscriptions itself
- the amount of files in the subscriptions repo
- the privacy implications – that might seem exaggerated, but IMHO we should take this into account. read on:
  - In „case one“ above – per-language files – the server that hosts the json files (currently github) will know which IP address has downloaded which language file.
- … (?)

---

After thinking about the aspects, I'd vote for **option two** – `official.json` and **one** L10n file. Below you can see how I would make `official.json` look like in this case:

``` json
{
  "l10n":{
    "serial":2014122300,
    "url":"https://…/subscriptions/master/locale.json"
  },
  "subscriptions":{
    "allow_embedded":{
      "serial":1329159661,
      "url":"https://…/subscriptions/master/official-allow_embedded.json",
      "title": "My title goes here",
      "description": "Description of allow_embedded…"
    }
  }
}
```

`locale.json` would look like:

``` json
{
  "fr":{
    "subscriptions":{
      "allow_embedded":{
        "title": "Translated title goes here",
        "description": "Translated description of allow_embedded…"
      }
    }
  },
  "de":{
    "and so on…":"…"
  }
}
```

---

When the specification is decided, RequestPolicy will have to download the additional file(s) and extract the added information.

**Comment by nodiscc** [comment]
Remaining todo: RequestPolicy will have to download the additional file(s) and extract the added information.

**Comment by myrdd** [comment]
Yes, this issue is blocked by #597, as the subscription module is currently badly extendable.
